spec/features/admin_categories_view_spec.rb

```diff
@@ -36,11 +35,11 @@
       expect(page).to have_content("New Category")
     end
     it "can delete a category" do
-       within('#category_vegan') do 
+       within('#category_vegan') do
```

Comment:
Use 2 (not 3) spaces for indentation.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.